src/auro-checkbox.scss

```diff
@@ -68,9 +65,10 @@ fieldset {
 }
 
 // refactor
+// This needs to be reviewed; is this being used?
```

Comment:
Can we make an issue for this instead of adding a TODO in the code?

Reply:
https://github.com/AlaskaAirlines/auro-checkbox/issues/40